src/EFTools/DesignXmlCore/VirtualTreeGrid/UI/VirtualTreeControl.cs

```diff
@@ -1899,12 +1899,7 @@ private void _WndProc(ref Message m)
                     {
                         var fContinue = false;
 
-#if DEBUG
-                        // - to not have to compile structs used only in Asserts
-                        Debug.Assert(Marshal.OffsetOf(typeof(NativeMethods.NMHDR), "code").ToInt32() == 8);
-                        Debug.Assert(Marshal.OffsetOf(typeof(NativeMethods.TOOLTIPTEXT), "lpszText").ToInt32() == 12);
-#endif
-                        var code = Marshal.ReadIntPtr(m.LParam, 8).ToInt32();
+                        var code = Marshal.ReadInt32(m.LParam, (int)Marshal.OffsetOf(typeof(NativeMethods.NMHDR), nameof(NativeMethods.NMHDR.code)));
```

Comment:
The ReadIntPtr call was never correct; this always should have been an int in the first place. Curiously, once the offset was fixed, this was then trying to read data that could no longer fit in an Int32.